.circleci/config.yml

```diff
@@ -340,6 +340,31 @@ jobs:
           command: yarn test -w 5 --shard "$(($CIRCLE_NODE_INDEX + 1))/$CIRCLE_NODE_TOTAL"
 
   test-e2e-front-admin:
+    docker:
+      - image: cimg/node:18.18
+        auth:
+          username: $DOCKER_HUB_USER
+          password: $DOCKER_HUB_PASSWORD
+    working_directory: ~/joanie/src/frontend/admin
+    resource_class: large
+    parallelism: 4
```

Comment:
Is it needed for the moment?

Reply:
not really but we're going to write a lot of component tests in the next PR